brewtils/models.py

```diff
@@ -1656,6 +1612,148 @@ def __repr__(self):
         )
 
 
+class User(BaseModel):
+    schema = "UserSchema"
+
+    def __init__(
+        self,
+        username=None,
+        id=None,
+        password=None,
+        roles=None,
+        local_roles=None,
+        upstream_roles=None,
+        user_alias_mapping=None,
+        metadata=None,
+        is_remote=False,
+        protected=False,
+        file_generated=False,
+    ):
+        self.id = id
+        self.username = username
```

Comment:
Id and username are flipped here. It would be nice if the arguments matched the assignment. Same with metadata and is_remote.